patches/minecraft/net/minecraft/client/network/play/ClientPlayNetHandler.java.patch

```diff
@@ -36,18 +36,15 @@
     }
  
     public void func_147279_a(SAnimateHandPacket p_147279_1_) {
-@@ -1004,8 +1008,10 @@
-          clientplayerentity1.func_233645_dx_().func_233784_a_(clientplayerentity.func_233645_dx_());
-       }
+@@ -1006,6 +1010,7 @@
  
-+      clientplayerentity1.updateSyncFields(clientplayerentity); // Forge: fix MC-10657
```

Comment:
Cool, this one is actually legit, old code to fix a bug that no longer is a bug.